package.json

```diff
@@ -5,13 +5,10 @@
   "homepage": ".",
   "devDependencies": {
     "@babel/traverse": "^7.16.3",
-    "@codemirror/language": "^0.19.5",
-    "@types/codemirror": "^5.60.5",
     "@types/i18n-js": "^3.8.2",
     "@typescript-eslint/eslint-plugin": "^4.5.0",
     "@typescript-eslint/parser": "^4.5.0",
     "autoprefixer": "^10.4.0",
-    "babel-eslint": "^10.1.0",
```

Comment:
why is this no longer needed?

Reply:
Babel-eslint is deprecated in favor of @babel/eslint-parser. I'm not entirely sure to what extent babel-eslint itself was being used, but I just noticed that message during the build-fixing process so I opted to remove it.